components/Landing/QuestChains.tsx

```diff
@@ -18,7 +18,7 @@ export const QuestChains: React.FC = () => {
       w="full"
       align="center"
       justify="center"
-      minH="80vh"
+      minH="100vh"
```

Comment:
we want this to be 80vh only, so that the whole illustration as well as a portion of the "what are we building" is seen. This is so that the user knows there is something to scroll down to.

Reply:
seems like a good idea! @davort ?

Reply:
however, i don't think we have to put too much effort/time into the upgrades of this site since we're planning to make a new animated landing page soon anyway.

Reply:
something like this could work as well: 
<img width="1603" alt="Screenshot 2022-10-30 at 11 32 42" src="https://user-images.githubusercontent.com/35112939/198874083-3bbf5b62-079a-4630-901f-c9f92aeaf234.png">